web-common/src/features/dashboards/time-dimension-details/util.ts

```diff
@@ -0,0 +1,18 @@
+// given area, return block to fetch
+export const getBlock = (blockSize: number, start: number, end: number) => {
```

Comment:
Adding a return type would both help the reader better understand what a "block" is and help IntelliSense where this function is consumed.